go/noise/noise.go

```diff
@@ -57,6 +59,22 @@ func ToKind(n Noise) Kind {
 	return GaussianNoise
 }
 
+// ConfidenceIntervalInt64 holds upper and lower bounds in int64 for the confidence interval.
+type ConfidenceIntervalInt64 struct {
+	LowerBound, UpperBound int64
+}
+
+// ConfidenceIntervalFloat64 holds upper and lower bounds in float64 for the confidence interval.
+type ConfidenceIntervalFloat64 struct {
+	LowerBound, UpperBound float64
+}
+
+// toConfidenceIntervalInt64 returns ConfidenceIntervalInt64 struct after rounding the upper
+// and lower bounds of the ConfidenceIntervalFloat64 struct.
+func (confInt ConfidenceIntervalFloat64) toConfidenceIntervalInt64() ConfidenceIntervalInt64 {
+	return ConfidenceIntervalInt64{int64(math.Round(confInt.LowerBound)), int64(math.Round(confInt.UpperBound))}
```

Comment:
ConfidenceIntervalInt64{Lower: int64(math.Round(confInt.LowerBound)), Upper: int64(math.Round(confInt.UpperBound))}

Without field names it will be error-prone, we can pass upper instead of lower and vice versa.